Cargo.toml

```diff
@@ -20,3 +20,4 @@ license = "MIT"
 [dependencies]
 thiserror = "^1.0"
 byteorder = "1"
+num-rational = "0.3"
```

Comment:
Just missed this. But this means `Cargo.lock` needs an update too, right?

Reply:
Yes, I missed it.
But, I am not sure if it is OK to keep `Cargo.lock`.
https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

Reply:
Ah, I wasn't aware it wasn't recommended for libraries. I've removed the Cargo.lock and added it to .gitignore. Thanks for the clarification.